packages/keystone/databaseAdapters/ReplicaKnexAdapter.js

```diff
@@ -0,0 +1,115 @@
+const { KnexAdapter } = require('@keystonejs/adapter-knex')
+const { knex } = require('knex')
+const { get, omit } = require('lodash')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+
+
+const logger = getLogger('replicaKnexAdapter')
+
+class ReplicaKnexAdapter extends KnexAdapter {
+    constructor (props) {
+        super(omit(props, ['connection']))
+        this.readConnection = get(props, ['connection', 'read'])
+        this.writeConnection = get(props, ['connection', 'write'])
+    }
+
+    async _connect () {
+        this.knex = knex({
```

Comment:
What's this is used for?

Reply:
This is main connection, that must be used for migrations applying and resolving final operation endpoint (connection)